shoop/front/settings.py

```diff
@@ -63,3 +63,11 @@
 #:
 #: Handled error cases are: 400, 403, 404, and 500
 SHOOP_FRONT_INSTALL_ERROR_HANDLERS = True
+
+#: A dictionary defining properties to override the default field properties in the
+#: checkout address form. For example, this can be used to set ``required`` to
+#: ``True`` to force a certain field to be required.
+#:
+#: The dictionary should map the field name (as a string) to a dictionary of the
+#: overridding properties.
+SHOOP_FRONT_ADDRESS_FIELD_PROPERTIES = {}
```

Comment:
Can you some this some good default. Like the fields we have now required? I think making some fields like city unrequired is a little bit hard without migrations. But maybe that is not part of this task.